### PR TITLE
fixed typing mistakes

### PR DESCRIPTION
**Pull request by navdeep300**
I have fixed a typing mistake. The mistake requires spacing between two different words. You can review the original file and the new file.

### CODE REVIEW

File: docs/scenarios/admin.rst [hunk 1]
```diff
@@ -244,7 +244,7 @@ Chef
 Puppet
 ------
 
-`Puppet <http://puppetlabs.com>`_ is IT Automation and configuration management
+`Puppet <http://puppetlabs.com>`_ is an IT Automation and configuration management
```
Comment (sigmavirus24) [inline docs/scenarios/admin.rst]: > Puppet is ... software from Puppet Labs that ...

In other words, `an` is incorrect here. Please remove this and update this PR.

File: docs/scenarios/admin.rst [hunk 2]
```diff
@@ -290,7 +290,7 @@ Here is an example of 'Hello World' in Puppet.
         #the notification message by default.
     }
 
-Here is another example with system based logic. Note how the operatingsystem
+Here is another example with system based logic. Note how the operating system
```
Comment (sigmavirus24) [inline docs/scenarios/admin.rst]: This is a good catch. When you update this, I'll merge this.

**Comment by navdeep300** [comment]
I have updated the file as suggested by you. You can see the changes and can merge into original one.

**Comment by sigmavirus24** [comment]
Merged with 769b74b147b6629c4dd460fa5811dd863079cbbb

**Pull request closed without merging**